samples/node-headless-ssr-proxy/config.js

```diff
@@ -141,25 +149,14 @@ const config = {
 
     if (cached) return Promise.resolve(cached);
 
-    return fetch(
```

Comment:
Looks like we can get rid of `node-fetch` dependency.

Reply:
It's true